src/store/metadata/stub/content/editions/4gc_2023-09.json

```diff
@@ -0,0 +1,112 @@
+{
+    "@context": "https://staging.idpd.uk/#ns",
+    "@id": "https://staging.idpd.uk/datasets/4gc/editions",
+    "@type": "hydra:Collection",
+    "title": "4G Coverage",
+    "editions": [
+        {
+            "@id": "https://staging.idpd.uk/datasets/4gc/editions/2023-09",
+            "@type": "dcat:Dataset",
+            "in_series": "https://staging.idpd.uk/datasets/4gc",
+            "identifier": "2023-09",
+            "title": "4G Coverage",
+            "summary": "Percentage of geographic areas with 4G signal outdoors from at least 1 operator (signal threshold: 105dBm), UK, September 2022.",
+            "description": "This dataset shows the percentage of geographic areas with 4G signal outdoors from at least 1 operator (signal threshold: 105dBm), in the UK, for September 2022.\n\n### Coverage \n\nUK\n\n#### Geography definition\n\nLocal Authority Districts and Unitary Authorities\n\n### Source\n\n - United Kingdom: [Ofcom](https://www.ofcom.org.uk/research-and-data/multi-sector-research/infrastructure-research/connected-nations-2022) \n\n\n",
+            "publisher": "https://staging.idpd.uk/office-for-national-statistics",
+            "creator": "ofcom",
+            "contact_point": {
+              "name": "Ofcom Open Data",
+              "email": "mailto:open.data@ofcom.org.uk"
+            },
+            "topics": [
+              "https://staging.idpd.uk/topics/itandinternetindustry"
+            ],
+            "frequency": "monthly",
+            "keywords": [
+              "broadband-mobile-coverage",
+              "connectivity",
+              "local-authority",
+              "subnational"],
+            "licence": "http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/",
+            "issued": "2023-03-31T12:10:00",
+            "modified": "2023-08-24T15:55:00",
+            "spatial_coverage": "K02000001",
+            "versions_url": "https://staging.idpd.uk/datasets/4gc/editions/2023-09/versions",
+            "versions": [
+              {
+                "@id": "https://staging.idpd.uk/datasets/4gc/editions/2023-09/versions/1",
+                "issued": "2023-08-24T09:30:00Z",
+                "modified": "2023-08-24T09:30:00Z"
+              }
+            ],
+            "temporal_coverage": "P1M",
```

Comment:
From the `datasets.json` the format needs to be
```json
           "temporal_coverage": {
              "start": "2022-09-01T00:00:00Z",
              "end": "2022-09-01T00:00:00Z"
            },
```

This format is the temporal_resolution. Each edition would need updated temporal_coverage.

Reply:
so I suspect these comments are sequencing things, he's working to the current examples and schemas and they're behind your corrections at tthe mo.

so its "right" in terms of the initial ask, but wrong in terms of latest (to be implemented) ask.

not sure best handling (beyond i get it done quicker), as its a slightly odd place where if he corrects to your steer all the validations will fail as the data is desycced from the current schema.

Reply:
update - in flight now, lets let it catch up then this works.

Reply:
Have updated